src/tests/Common/CLRTest.Execute.Batch.targets

```diff
@@ -318,6 +321,11 @@ set CLRTestExitCode=!ERRORLEVEL!
 if defined RunCrossGen (
   call :ReleaseLock
 )
+if defined RunNativeAot (
+  if not defined CLRTestNoCleanup (
+    IF EXIST native rmdir /s /q native
```

Comment:
Why do we need this just in the Windows version of the script and how does it play together with similar clauses in the execution script snippet generator in <code>CLRTest.NativeAOT.targets</code>?

Reply:
Ah, good catch. I guess we never run enough tests on Unix systems where not deleting the results would be a problem. I'll fix that.